content/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config.md

```diff
@@ -0,0 +1,456 @@
+---
+title: Encryptor config
+weight: 1
+---
+
+# Encryptor config
+
+This config is responsible for configuration of proactive security controls of AcraServer and can be specified
+via `--encryptor_config=<path>` CLI parameter or `encryptor_config` key in YAML configuration file for AcraServer.
+
+The following security controls can be configured:
+- Transparent encryption
+- Searchable encryption
+- Masking
+- Tokenization
+
+
+There is full example of configuration file with all options:
+
+```
+defaults:
+  crypto_envelope: "<acrablock|acrastruct>" # [optional] [default=acrablock] 
+  reencrypting_to_acrablocks: false # [optional] [default=false]
```

Comment:
To avoid horizontal scroll in config example, I would recommend making it like this.
These two lines have no issues, but we got much longer lines below, better to split them, so split these as well.
```suggestion
  # [optional] [default=acrablock] 
  crypto_envelope: "<acrablock|acrastruct>"
  # [optional] [default=false]
  reencrypting_to_acrablocks: false
```